util/pkg/vfs/s3fs.go

```diff
@@ -106,6 +106,71 @@ func (p *S3Path) Remove() error {
 	return nil
 }
 
+func (p *S3Path) RemoveAll() error {
+	client, err := p.client()
+	if err != nil {
+		return err
+	}
+
+	klog.V(8).Infof("removing file %s", p)
```

Comment:
nit: "removing all versions of %s" or "remove file %s (all versions)" might be clearer